### ISSUE TITLE
Page guide.

### ISSUE THREAD

**Issue by hifall** [vue-bulma/vue-admin#152]
Page guides are becoming popular these days in non-trivial web apps. A nice page guide can onboard first-time users quickly. An example jQuery page guide is here: [TraceLytics Page Guide](http://tracelytics.github.io/pageguide/).

Would adding a page guide to vue-admin be considered?

Thanks.

**Comment by luventa** [comment]
hmm...Yup I guess we may need this.

**Comment by Okdesk** [comment]
Thanks for your kind reply.

Actually I guess the value of a page guide (or page tour, etc.) is that it walks the new user through the app WITHOUT requiring the user to read extra docs, so that the user can onboard really quickly. Nowadays there are maybe thousands of web apps out there, to say the least, so it's important to walk the user through as quickly as possible, in order to retain the user.

Compared to out-of-app docs like Wiki, I think a page guide offers these advantages (at least):
1 in-app: the instructions are right within the app, with appropriate context info; and users can learn by examples;
2 concise: the text in page guide is often shorter than in normal docs, imposing minimal cognitive burden.

**Comment by hifall** [comment]
Thanks for your kind reply.

Actually I guess the value of a page guide (or page tour, etc.) is that it walks the new user through the app WITHOUT requiring the user to read extra docs, so that the user can onboard really quickly. Nowadays there are maybe thousands of web apps out there, to say the least, so it's important to walk the user through as quickly as possible, in order to retain the user.

Compared to out-of-app docs like Wiki, I think a page guide offers these advantages (at least):
1 in-app: the instructions are right within the app, with appropriate context info; and users can learn by examples;
2 concise: the text in page guide is often shorter than in normal docs, imposing minimal cognitive burden.

In short, a page guide can be viewed as a quick-and-light document in the SaaS age, while the traditional documents are slow and heavy.
